plugins/org.locationtech.udig.catalog.db2/src/org/locationtech/udig/catalog/internal/db2/ui/DB2WizardPage.java

```diff
@@ -48,11 +47,12 @@
  * </p>
  * 
  * @author Justin Deoliveira, jdeolive, for Refractions Research, Inc.
- * @author dadler
  * @author Jesse Eichar, jeichar, for Refractions Research, Inc.
  * @author Richard Gould, rgould, for Refractions Research, Inc.
  * @author Adrian Custer, acuster.
  * @since 1.0.1
+ * @author David Adler, dadler, for AdtechGeospatial 
+ * @since 2.0.x 
```

Comment:
Target release is 2.x, I guess its better to track changes for a release in locationtech bugzilla (for the monent), I filed an enhancement for your maintanance work : https://locationtech.org/bugs/show_bug.cgi?id=98